app/src/main/java/org/mozilla/rocket/privately/browse/BrowserFragment.kt

```diff
@@ -249,6 +249,7 @@ class BrowserFragment : LocaleAwareFragment(),
 
         sessionManager.dropTab(focus.id)
         ScreenNavigator.get(activity).popToHomeScreen(true)
+        listener?.onNotified(this, TYPE.DROP_BROWSING_PAGES, null)
```

Comment:
Consider writing a function sanitize that reuses onDeleteClicked() or call onDeleteClicked itself directly.

Reply:
More refactoring coming up. Will do that there.